imjoy/imjoyPluginEngine.py

```diff
@@ -634,7 +634,7 @@ def runCmd(
     return output
 
 
-def parseEnv(env, work_dir):
+def parseEnv(env, work_dir, pname):
```

Comment:
maybe change `pname` to something like `default_env_name`, and when call this function use `default_env_name= pname + '-' +tag if tag != '' else pname`?

Reply:
Ok.